server/config.py

```diff
@@ -28,6 +28,30 @@ class EnvironmentConfig:
     # A freely definable secret key for connecting the front end with the back end
     SECRET_KEY = os.getenv("TM_SECRET", None)
 
+    ML_ENABLER_URL = os.getenv("TM_ML_ENABLER_URL", "https://ml-enabler.hotosm.org/v1")
```

Comment:
Can we have a config file without a fallback to our resources? Ideally we specify them in the config file ourselves, but we don't want to expose them/